### DISCUSSION TITLE
Hypothesidian - A script to retrieve annotations from Hypothes.is into Obsidian

### DISCUSSION THREAD

**Discussion by TfTHacker** [SilentVoid13/Templater#191] [Templates Showcase]
Script here:

https://gist.github.com/tfthacker/c48bca69f1520deed0ecbc8840f6241a

![image](https://user-images.githubusercontent.com/69121180/116994975-52d5aa00-ac8e-11eb-95ac-27c2ed1b49c0.png)

**Comment by zzdcgg** [comment]
Good job RoamHacker!

**Comment by natarslan** [comment]
This is great! Do you think it's possible to get annotations by a specific tag?

**Comment by cs-mshah** [comment]
The link is broken

**Reply by TfTHacker** [reply]
Sorry about that. Just updated the link

**Comment by AB1908** [comment]
Looks good but what have you got against the Hypothesis plugin's dev? 🤣
